diff --git a/src/plugins/output/NDI/CMakeLists.txt b/src/plugins/output/NDI/CMakeLists.txt
index e24103ab..a0c6f1e2 100644
--- a/src/plugins/output/NDI/CMakeLists.txt
+++ b/src/plugins/output/NDI/CMakeLists.txt
@@ -11,15 +11,10 @@ SET(_target
     "NDI"
 )
 
 SET(_sources
     NDIModule.cpp NDIVideoDevice.cpp NDIOutput.cpp
 )
 
-ADD_LIBRARY(
-  ${_target} SHARED
-  ${_sources}
-)
-
 # Note that a super project, a project consuming Open RV as a submodule, could decide to fetch the NDI_SDK::NDI_SDK from artifactory and thus have already
 # created the NDI_SDK::NDI_SDK target. In that case, we should not try to find.
 IF(NOT TARGET NDI_SDK::NDI_SDK)
@@ -51,6 +46,11 @@ IF(NOT TARGET NDI_SDK::NDI_SDK)
   ENDIF()
 ENDIF()
 
+ADD_LIBRARY(
+  ${_target} SHARED
+  ${_sources}
+)
+
 TARGET_INCLUDE_DIRECTORIES(
   ${_target}
   PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}
