diff --git a/cmake/dependencies/glew.cmake b/cmake/dependencies/glew.cmake
index 7b665a4..f2c33c8 100644
--- a/cmake/dependencies/glew.cmake
+++ b/cmake/dependencies/glew.cmake
@@ -27,6 +27,22 @@ SET(_glew_lib
     ${_lib_dir}/${_glew_lib_name}
 )
-
-EXTERNALPROJECT_ADD(
+SET(_glew_make_options)
+IF(RV_TARGET_LINUX)
+  LIST(
+    APPEND
+    _glew_make_options
+    "PYTHON=$ENV{BUILD_PREFIX}/bin/python"
+    "CFLAGS.EXTRA=-fPIC -Wcast-qual -ansi -pedantic -fno-stack-protector -I$ENV{PREFIX}/include"
+    "LDFLAGS.EXTRA=-L$ENV{PREFIX}/lib"
+  )
+ELSE()
+  LIST(
+    APPEND
+    _glew_make_options
+    "PYTHON=$ENV{BUILD_PREFIX}/bin/python"
+  )
+ENDIF()
+
+EXTERNALPROJECT_ADD(
   ${_target}
   SOURCE_DIR ${RV_DEPS_BASE_DIR}/${_target}/src
@@ -35,8 +51,8 @@ EXTERNALPROJECT_ADD(
   URL_MD5 ${_download_hash}
   DOWNLOAD_NAME glew-glew-${_version}.tar.gz
   DOWNLOAD_DIR ${RV_DEPS_DOWNLOAD_DIR}
-  CONFIGURE_COMMAND cd auto && ${_make_command} && cd .. && ${_make_command}
-  BUILD_COMMAND ${_make_command} -j${_cpu_count} GLEW_DEST=${_install_dir}
+  CONFIGURE_COMMAND cd auto && ${_make_command} ${_glew_make_options} && cd .. && ${_make_command} ${_glew_make_options}
+  BUILD_COMMAND ${_make_command} -j${_cpu_count} GLEW_DEST=${_install_dir} ${_glew_make_options}
   INSTALL_COMMAND ${_make_command} install LIBDIR=${_lib_dir} GLEW_DEST=${_install_dir}
   BUILD_IN_SOURCE TRUE
   BUILD_ALWAYS FALSE
