diff --git a/cmake/dependencies/ocio.cmake b/cmake/dependencies/ocio.cmake
--- a/cmake/dependencies/ocio.cmake
+++ b/cmake/dependencies/ocio.cmake
@@ -51,7 +51,11 @@ IF(RV_TARGET_WINDOWS)
   SET(_ocio_win_sharedlib_path
       ${_bin_dir}/${_ocio_win_sharedlibname}
   )
-  LIST(APPEND _byproducts ${_ocio_win_sharedlib_path})
+  SET(_libpath
+      ${_ocio_win_sharedlib_path}
+  )
+  LIST(APPEND _byproducts ${_ocio_win_sharedlib_path})
+  LIST(APPEND _byproducts ${_implibpath})
 ENDIF()
 
 IF(RV_TARGET_WINDOWS)
