diff --git a/cmake/dependencies/oiio.cmake b/cmake/dependencies/oiio.cmake
--- a/cmake/dependencies/oiio.cmake
+++ b/cmake/dependencies/oiio.cmake
@@ -136,6 +136,7 @@ IF(NOT RV_TARGET_LINUX)
   # Linux has a Link error related to relocation; WebP appears not built with -fPIC. Hence OIIO will build WebP itself.
 ENDIF()
 LIST(APPEND _configure_options "-DZLIB_ROOT=${RV_DEPS_ZLIB_ROOT_DIR}")
+LIST(APPEND _configure_options "-Dfmt_ROOT=$ENV{PREFIX}")
-
+
 # OIIO tools are not needed.
 LIST(APPEND _configure_options "-DOIIO_BUILD_TOOLS=OFF" "-DOIIO_BUILD_TESTS=OFF")
