diff --git a/src/build/make_openssl.py b/src/build/make_openssl.py
index 000000000..000000000 100644
--- a/src/build/make_openssl.py
+++ b/src/build/make_openssl.py
@@ -187,6 +187,7 @@ def configure() -> None:
 
     configure_args.append(f"--prefix={OUTPUT_DIR}")
     configure_args.append(f"--openssldir={OUTPUT_DIR}")
+    configure_args.append("--libdir=lib")
 
     # Skip building the test suite
     configure_args.append("no-tests")
