diff --git a/cmake/dependencies/openssl.cmake b/cmake/dependencies/openssl.cmake
--- a/cmake/dependencies/openssl.cmake
+++ b/cmake/dependencies/openssl.cmake
@@ -196,6 +196,11 @@ ELSE()
     )
   ENDIF()
 
+  SET(_openssl_byproducts ${_crypto_lib} ${_ssl_lib})
+  IF(RV_TARGET_WINDOWS)
+    LIST(APPEND _openssl_byproducts ${_implibpath_crypto} ${_implibpath_ssl})
+  ENDIF()
+
   EXTERNALPROJECT_ADD(
     ${_target}
     DOWNLOAD_NAME ${_target}_${_version}.zip
@@ -211,7 +217,7 @@ ELSE()
     INSTALL_COMMAND ${_make_command} --install
     BUILD_IN_SOURCE TRUE
     BUILD_ALWAYS FALSE
-    BUILD_BYPRODUCTS ${_crypto_lib} ${_ssl_lib}
+    BUILD_BYPRODUCTS ${_openssl_byproducts}
     USES_TERMINAL_BUILD TRUE
   )
 
