--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,6 +75,12 @@
 
 MESSAGE("PROJECT DIR ${PROJECT_SOURCE_DIR}")
 
+OPTION(
+  RV_BUILD_OSMESA
+  "Build Linux OSMesa-based offscreen rendering support."
+  ON
+)
+
 # Specify our own CMake modules so we can include them right after.
 SET(CMAKE_MODULE_PATH
     ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake/defaults ${PROJECT_SOURCE_DIR}/cmake/dependencies ${PROJECT_SOURCE_DIR}/cmake/globals
--- a/src/bin/imgtools/CMakeLists.txt
+++ b/src/bin/imgtools/CMakeLists.txt
@@ -6,7 +6,7 @@
 ADD_SUBDIRECTORY(rmsImageDiff)
 ADD_SUBDIRECTORY(rvio)
 
-IF(RV_TARGET_LINUX)
+IF(RV_TARGET_LINUX AND RV_BUILD_OSMESA)
   ADD_SUBDIRECTORY(rvio_sw)
 ENDIF()
 
--- a/src/lib/graphics/CMakeLists.txt
+++ b/src/lib/graphics/CMakeLists.txt
@@ -8,6 +8,6 @@
   ADD_SUBDIRECTORY(TwkGLFCoreGraphics)
 ENDIF()
 ADD_SUBDIRECTORY(TwkGLFFBO)
-IF(RV_TARGET_LINUX)
+IF(RV_TARGET_LINUX AND RV_BUILD_OSMESA)
   ADD_SUBDIRECTORY(TwkGLFMesa)
 ENDIF()
--- a/src/lib/image/CMakeLists.txt
+++ b/src/lib/image/CMakeLists.txt
@@ -19,7 +19,7 @@
 ADD_SUBDIRECTORY(MovieMuDraw)
 ADD_SUBDIRECTORY(MovieRV_FBO)
-IF(RV_TARGET_LINUX)
+IF(RV_TARGET_LINUX AND RV_BUILD_OSMESA)
   ADD_SUBDIRECTORY(MovieRV)
 ENDIF()
 ADD_SUBDIRECTORY(MovieStdinFB)
