diff --git a/src/lib/network/CMakeLists.txt b/src/lib/network/CMakeLists.txt
index c4d12cb1..4e9c37f8 100644
--- a/src/lib/network/CMakeLists.txt
+++ b/src/lib/network/CMakeLists.txt
@@ -3,5 +3,10 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+OPTION(RV_BUILD_QTBONJOUR "Build Bonjour service discovery helper" ON)
+
 ADD_SUBDIRECTORY(TwkQtChat)
-ADD_SUBDIRECTORY(QtBonjour)
+
+IF(RV_BUILD_QTBONJOUR)
+  ADD_SUBDIRECTORY(QtBonjour)
+ENDIF()
