diff --git a/src/bin/python/py-interp/CMakeLists.txt b/src/bin/python/py-interp/CMakeLists.txt
--- a/src/bin/python/py-interp/CMakeLists.txt
+++ b/src/bin/python/py-interp/CMakeLists.txt
@@ -52,10 +52,20 @@

 RV_STAGE(TYPE "EXECUTABLE_WITH_PLUGINS" TARGET ${_target})

+SET(_py_interp_qt_env QT_QPA_PLATFORM=minimal)
+IF(RV_TARGET_WINDOWS)
+  LIST(
+    APPEND
+    _py_interp_qt_env
+    QT_PLUGIN_PATH=${RV_STAGE_LIB_DIR}/site-packages/PySide6/plugins
+    QT_QPA_PLATFORM_PLUGIN_PATH=${RV_STAGE_LIB_DIR}/site-packages/PySide6/plugins/platforms
+  )
+ENDIF()
+
 ADD_CUSTOM_COMMAND(
   COMMENT "Generating .pyc for ${RV_STAGE_PLUGINS_PYTHON_DIR}"
   OUTPUT ${RV_STAGE_PLUGINS_PYTHON_DIR}/.compiled
-  COMMAND ${CMAKE_COMMAND} -E env QT_QPA_PLATFORM=minimal $<TARGET_FILE:${_target}> -m compileall -f ${RV_STAGE_PLUGINS_PYTHON_DIR}
+  COMMAND ${CMAKE_COMMAND} -E env ${_py_interp_qt_env} $<TARGET_FILE:${_target}> -m compileall -f ${RV_STAGE_PLUGINS_PYTHON_DIR}
   COMMAND ${CMAKE_COMMAND} -E touch ${RV_STAGE_PLUGINS_PYTHON_DIR}/.compiled
   DEPENDS ${_target} python_source_modules installed_packages
 )
@@ -71,5 +81,5 @@ ADD_DEPENDENCIES(compiled_python_source_modules pyc_generation)
 ADD_TEST(
   NAME "py-interp ${python_test_file_name}"
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMAND ${CMAKE_COMMAND} -E env QT_QPA_PLATFORM=minimal "$<TARGET_FILE:${_target}>" -m unittest --verbose
+  COMMAND ${CMAKE_COMMAND} -E env ${_py_interp_qt_env} "$<TARGET_FILE:${_target}>" -m unittest --verbose
 )
