diff --git a/src/build/test_python.py b/src/build/test_python.py
--- a/src/build/test_python.py
+++ b/src/build/test_python.py
@@ -60,11 +60,6 @@
             "-c",
             "\n".join(
                 [
-                    # Check for tkinter
-                    "try:",
-                    "    import tkinter",
-                    "except Exception:",
-                    "    import Tkinter as tkinter",
                     # Make sure certifi is available
                     "import certifi",
                     # Make sure the SSL_CERT_FILE variable is set
