diff --git i/src/lib/mu/MuQt6/CMakeLists.txt w/src/lib/mu/MuQt6/CMakeLists.txt
index 71144dc..d326707 100644
--- i/src/lib/mu/MuQt6/CMakeLists.txt
+++ w/src/lib/mu/MuQt6/CMakeLists.txt
@@ -285,7 +285,7 @@ ENDFOREACH()
 
 ADD_LIBRARY(
   ${_target} STATIC
-  ${_sources} ${CMAKE_CURRENT_SOURCE_DIR}/${_target}/generated/moc_SignalSpy_filtered.hpp
+  ${_sources}
 )
 
 #
diff --git i/src/lib/mu/MuQt6/MuQt6/SignalSpy.h w/src/lib/mu/MuQt6/MuQt6/SignalSpy.h
index df67fd5..61b3331 100644
--- i/src/lib/mu/MuQt6/MuQt6/SignalSpy.h
+++ w/src/lib/mu/MuQt6/MuQt6/SignalSpy.h
@@ -16,7 +16,6 @@ namespace Mu
 
     class SignalSpy : public QSignalSpy
     {
-        Q_OBJECT
 
     public:
         enum Types
@@ -43,6 +42,7 @@ namespace Mu
 
         virtual ~SignalSpy();
 
+        int qt_metacall(QMetaObject::Call, int, void**);
         int original_qt_metacall(QMetaObject::Call, int, void**);
 
     private:
diff --git i/src/lib/mu/MuQt6/QBrushType.cpp w/src/lib/mu/MuQt6/QBrushType.cpp
index 0cbc3e7..72061ca 100644
--- i/src/lib/mu/MuQt6/QBrushType.cpp
+++ w/src/lib/mu/MuQt6/QBrushType.cpp
@@ -261,7 +261,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QBrush& arg0 = getqtype<QBrushType>(param_this);
         const QBrush arg1 = getqtype<QBrushType>(param_brush);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QBrush_operatorEQ_EQ__bool_QBrush_QBrush(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_brush)
@@ -269,7 +269,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QBrush& arg0 = getqtype<QBrushType>(param_this);
         const QBrush arg1 = getqtype<QBrushType>(param_brush);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QBrush0, Pointer)
diff --git i/src/lib/mu/MuQt6/QByteArrayType.cpp w/src/lib/mu/MuQt6/QByteArrayType.cpp
index 03c979b..1447a90 100644
--- i/src/lib/mu/MuQt6/QByteArrayType.cpp
+++ w/src/lib/mu/MuQt6/QByteArrayType.cpp
@@ -251,7 +251,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QByteArray& arg0 = getqtype<QByteArrayType>(param_this);
         const QString arg1 = qstring(param_str);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     Pointer qt_QByteArray_operatorPlus_EQ__QByteArray_QByteArray_QByteArray(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_ba)
@@ -267,7 +267,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QByteArray& arg0 = getqtype<QByteArrayType>(param_this);
         const QString arg1 = qstring(param_str);
-        return arg0.operator<(arg1);
+        return arg0 < arg1;
     }
 
     bool qt_QByteArray_operatorLT_EQ__bool_QByteArray_string(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_str)
@@ -275,7 +275,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QByteArray& arg0 = getqtype<QByteArrayType>(param_this);
         const QString arg1 = qstring(param_str);
-        return arg0.operator<=(arg1);
+        return arg0 <= arg1;
     }
 
     bool qt_QByteArray_operatorEQ_EQ__bool_QByteArray_string(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_str)
@@ -283,7 +283,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QByteArray& arg0 = getqtype<QByteArrayType>(param_this);
         const QString arg1 = qstring(param_str);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     bool qt_QByteArray_operatorGT__bool_QByteArray_string(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_str)
@@ -291,7 +291,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QByteArray& arg0 = getqtype<QByteArrayType>(param_this);
         const QString arg1 = qstring(param_str);
-        return arg0.operator>(arg1);
+        return arg0 > arg1;
     }
 
     bool qt_QByteArray_operatorGT_EQ__bool_QByteArray_string(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_str)
@@ -299,7 +299,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QByteArray& arg0 = getqtype<QByteArrayType>(param_this);
         const QString arg1 = qstring(param_str);
-        return arg0.operator>=(arg1);
+        return arg0 >= arg1;
     }
 
     Pointer qt_QByteArray_fromBase64_QByteArray_QByteArray_int(Mu::Thread& NODE_THREAD, Pointer param_base64, int param_options)
diff --git i/src/lib/mu/MuQt6/QColorType.cpp w/src/lib/mu/MuQt6/QColorType.cpp
index 6aac945..f3fa9c0 100644
--- i/src/lib/mu/MuQt6/QColorType.cpp
+++ w/src/lib/mu/MuQt6/QColorType.cpp
@@ -625,7 +625,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QColor arg0 = getqtype<QColorType>(param_this);
         const QColor arg1 = getqtype<QColorType>(param_color);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QColor_operatorEQ_EQ__bool_QColor_QColor(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_color)
@@ -633,7 +633,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QColor arg0 = getqtype<QColorType>(param_this);
         const QColor arg1 = getqtype<QColorType>(param_color);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QColor_colorNames_stringBSB_ESB_(Mu::Thread& NODE_THREAD)
diff --git i/src/lib/mu/MuQt6/QDirType.cpp w/src/lib/mu/MuQt6/QDirType.cpp
index 81c0b45..292d67c 100644
--- i/src/lib/mu/MuQt6/QDirType.cpp
+++ w/src/lib/mu/MuQt6/QDirType.cpp
@@ -401,7 +401,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QDir& arg0 = getqtype<QDirType>(param_this);
         const QDir arg1 = getqtype<QDirType>(param_dir);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QDir_operatorEQ_EQ__bool_QDir_QDir(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_dir)
@@ -409,7 +409,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QDir& arg0 = getqtype<QDirType>(param_this);
         const QDir arg1 = getqtype<QDirType>(param_dir);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     void qt_QDir_addSearchPath_void_string_string(Mu::Thread& NODE_THREAD, Pointer param_prefix, Pointer param_path)
diff --git i/src/lib/mu/MuQt6/QFontType.cpp w/src/lib/mu/MuQt6/QFontType.cpp
index f709474..4d6c3dd 100644
--- i/src/lib/mu/MuQt6/QFontType.cpp
+++ w/src/lib/mu/MuQt6/QFontType.cpp
@@ -537,7 +537,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QFont& arg0 = getqtype<QFontType>(param_this);
         const QFont arg1 = getqtype<QFontType>(param_f);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QFont_operatorLT__bool_QFont_QFont(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_f)
@@ -545,7 +545,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QFont& arg0 = getqtype<QFontType>(param_this);
         const QFont arg1 = getqtype<QFontType>(param_f);
-        return arg0.operator<(arg1);
+        return arg0 < arg1;
     }
 
     bool qt_QFont_operatorEQ_EQ__bool_QFont_QFont(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_f)
@@ -553,7 +553,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QFont& arg0 = getqtype<QFontType>(param_this);
         const QFont arg1 = getqtype<QFontType>(param_f);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     void qt_QFont_insertSubstitution_void_string_string(Mu::Thread& NODE_THREAD, Pointer param_familyName, Pointer param_substituteName)
diff --git i/src/lib/mu/MuQt6/QGradientType.cpp w/src/lib/mu/MuQt6/QGradientType.cpp
index ac282ae..8b01973 100644
--- i/src/lib/mu/MuQt6/QGradientType.cpp
+++ w/src/lib/mu/MuQt6/QGradientType.cpp
@@ -134,7 +134,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QGradient& arg0 = getqtype<QGradientType>(param_this);
         const QGradient arg1 = getqtype<QGradientType>(param_gradient);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QGradient_operatorEQ_EQ__bool_QGradient_QGradient(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_gradient)
@@ -142,7 +142,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QGradient& arg0 = getqtype<QGradientType>(param_this);
         const QGradient arg1 = getqtype<QGradientType>(param_gradient);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QGradient0, Pointer)
diff --git i/src/lib/mu/MuQt6/QHostAddressType.cpp w/src/lib/mu/MuQt6/QHostAddressType.cpp
index 9acd74d..d6f1cd6 100644
--- i/src/lib/mu/MuQt6/QHostAddressType.cpp
+++ w/src/lib/mu/MuQt6/QHostAddressType.cpp
@@ -239,7 +239,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QHostAddress arg0 = getqtype<QHostAddressType>(param_this);
         const QHostAddress arg1 = getqtype<QHostAddressType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QHostAddress_operatorBang_EQ__bool_QHostAddress_int(Mu::Thread& NODE_THREAD, Pointer param_this, int param_other)
@@ -247,7 +247,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QHostAddress arg0 = getqtype<QHostAddressType>(param_this);
         QHostAddress::SpecialAddress arg1 = (QHostAddress::SpecialAddress)(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QHostAddress_operatorEQ_EQ__bool_QHostAddress_QHostAddress(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -255,7 +255,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QHostAddress arg0 = getqtype<QHostAddressType>(param_this);
         const QHostAddress arg1 = getqtype<QHostAddressType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     bool qt_QHostAddress_operatorEQ_EQ__bool_QHostAddress_int(Mu::Thread& NODE_THREAD, Pointer param_this, int param_other)
@@ -263,7 +263,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QHostAddress arg0 = getqtype<QHostAddressType>(param_this);
         QHostAddress::SpecialAddress arg1 = (QHostAddress::SpecialAddress)(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QHostAddress0, Pointer)
diff --git i/src/lib/mu/MuQt6/QImageType.cpp w/src/lib/mu/MuQt6/QImageType.cpp
index 2101b6c..af165f5 100644
--- i/src/lib/mu/MuQt6/QImageType.cpp
+++ w/src/lib/mu/MuQt6/QImageType.cpp
@@ -622,7 +622,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QImage& arg0 = getqtype<QImageType>(param_this);
         const QImage arg1 = getqtype<QImageType>(param_image);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QImage_operatorEQ_EQ__bool_QImage_QImage(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_image)
@@ -630,7 +630,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QImage& arg0 = getqtype<QImageType>(param_this);
         const QImage arg1 = getqtype<QImageType>(param_image);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QImage_trueMatrix_QTransform_QTransform_int_int(Mu::Thread& NODE_THREAD, Pointer param_matrix, int param_width,
diff --git i/src/lib/mu/MuQt6/QItemSelectionRangeType.cpp w/src/lib/mu/MuQt6/QItemSelectionRangeType.cpp
index 834b436..ff3e34e 100644
--- i/src/lib/mu/MuQt6/QItemSelectionRangeType.cpp
+++ w/src/lib/mu/MuQt6/QItemSelectionRangeType.cpp
@@ -232,7 +232,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QItemSelectionRange arg0 = getqtype<QItemSelectionRangeType>(param_this);
         const QItemSelectionRange arg1 = getqtype<QItemSelectionRangeType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QItemSelectionRange_operatorEQ_EQ__bool_QItemSelectionRange_QItemSelectionRange(Mu::Thread& NODE_THREAD, Pointer param_this,
@@ -241,7 +241,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QItemSelectionRange arg0 = getqtype<QItemSelectionRangeType>(param_this);
         const QItemSelectionRange arg1 = getqtype<QItemSelectionRangeType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QItemSelectionRange0, Pointer)
diff --git i/src/lib/mu/MuQt6/QKeySequenceType.cpp w/src/lib/mu/MuQt6/QKeySequenceType.cpp
index b899384..5b22b35 100644
--- i/src/lib/mu/MuQt6/QKeySequenceType.cpp
+++ w/src/lib/mu/MuQt6/QKeySequenceType.cpp
@@ -151,7 +151,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QKeySequence& arg0 = getqtype<QKeySequenceType>(param_this);
         const QKeySequence arg1 = getqtype<QKeySequenceType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QKeySequence_operatorLT__bool_QKeySequence_QKeySequence(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -159,7 +159,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QKeySequence& arg0 = getqtype<QKeySequenceType>(param_this);
         const QKeySequence arg1 = getqtype<QKeySequenceType>(param_other);
-        return arg0.operator<(arg1);
+        return arg0 < arg1;
     }
 
     bool qt_QKeySequence_operatorLT_EQ__bool_QKeySequence_QKeySequence(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -167,7 +167,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QKeySequence& arg0 = getqtype<QKeySequenceType>(param_this);
         const QKeySequence arg1 = getqtype<QKeySequenceType>(param_other);
-        return arg0.operator<=(arg1);
+        return arg0 <= arg1;
     }
 
     bool qt_QKeySequence_operatorEQ_EQ__bool_QKeySequence_QKeySequence(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -175,7 +175,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QKeySequence& arg0 = getqtype<QKeySequenceType>(param_this);
         const QKeySequence arg1 = getqtype<QKeySequenceType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     bool qt_QKeySequence_operatorGT__bool_QKeySequence_QKeySequence(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -183,7 +183,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QKeySequence& arg0 = getqtype<QKeySequenceType>(param_this);
         const QKeySequence arg1 = getqtype<QKeySequenceType>(param_other);
-        return arg0.operator>(arg1);
+        return arg0 > arg1;
     }
 
     bool qt_QKeySequence_operatorGT_EQ__bool_QKeySequence_QKeySequence(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -191,7 +191,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QKeySequence& arg0 = getqtype<QKeySequenceType>(param_this);
         const QKeySequence arg1 = getqtype<QKeySequenceType>(param_other);
-        return arg0.operator>=(arg1);
+        return arg0 >= arg1;
     }
 
     Pointer qt_QKeySequence_fromString_QKeySequence_string_int(Mu::Thread& NODE_THREAD, Pointer param_str, int param_format)
diff --git i/src/lib/mu/MuQt6/QMatrix4x4Type.cpp w/src/lib/mu/MuQt6/QMatrix4x4Type.cpp
index 490bbcf..a308536 100644
--- i/src/lib/mu/MuQt6/QMatrix4x4Type.cpp
+++ w/src/lib/mu/MuQt6/QMatrix4x4Type.cpp
@@ -381,7 +381,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QMatrix4x4 arg0 = getqtype<QMatrix4x4Type>(param_this);
         const QMatrix4x4 arg1 = getqtype<QMatrix4x4Type>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     Pointer qt_QMatrix4x4_operatorPlus_EQ__QMatrix4x4_QMatrix4x4_QMatrix4x4(Mu::Thread& NODE_THREAD, Pointer param_this,
@@ -415,7 +415,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QMatrix4x4 arg0 = getqtype<QMatrix4x4Type>(param_this);
         const QMatrix4x4 arg1 = getqtype<QMatrix4x4Type>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QMatrix4x40, Pointer)
diff --git i/src/lib/mu/MuQt6/QModelIndexType.cpp w/src/lib/mu/MuQt6/QModelIndexType.cpp
index 7bff1f9..0395ed3 100644
--- i/src/lib/mu/MuQt6/QModelIndexType.cpp
+++ w/src/lib/mu/MuQt6/QModelIndexType.cpp
@@ -160,7 +160,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QModelIndex arg0 = getqtype<QModelIndexType>(param_this);
         const QModelIndex arg1 = getqtype<QModelIndexType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QModelIndex_operatorLT__bool_QModelIndex_QModelIndex(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -168,7 +168,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QModelIndex arg0 = getqtype<QModelIndexType>(param_this);
         const QModelIndex arg1 = getqtype<QModelIndexType>(param_other);
-        return arg0.operator<(arg1);
+        return arg0 < arg1;
     }
 
     bool qt_QModelIndex_operatorEQ_EQ__bool_QModelIndex_QModelIndex(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -176,7 +176,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QModelIndex arg0 = getqtype<QModelIndexType>(param_this);
         const QModelIndex arg1 = getqtype<QModelIndexType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QModelIndex0, Pointer)
diff --git i/src/lib/mu/MuQt6/QNetworkCookieType.cpp w/src/lib/mu/MuQt6/QNetworkCookieType.cpp
index 2670200..d2929aa 100644
--- i/src/lib/mu/MuQt6/QNetworkCookieType.cpp
+++ w/src/lib/mu/MuQt6/QNetworkCookieType.cpp
@@ -260,7 +260,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QNetworkCookie& arg0 = getqtype<QNetworkCookieType>(param_this);
         const QNetworkCookie arg1 = getqtype<QNetworkCookieType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QNetworkCookie_operatorEQ_EQ__bool_QNetworkCookie_QNetworkCookie(Mu::Thread& NODE_THREAD, Pointer param_this,
@@ -269,7 +269,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QNetworkCookie& arg0 = getqtype<QNetworkCookieType>(param_this);
         const QNetworkCookie arg1 = getqtype<QNetworkCookieType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QNetworkCookie_parseCookies_qt__QNetworkCookieBSB_ESB__QByteArray(Mu::Thread& NODE_THREAD, Pointer param_cookieString)
diff --git i/src/lib/mu/MuQt6/QPaletteType.cpp w/src/lib/mu/MuQt6/QPaletteType.cpp
index cb5644b..5b29032 100644
--- i/src/lib/mu/MuQt6/QPaletteType.cpp
+++ w/src/lib/mu/MuQt6/QPaletteType.cpp
@@ -432,7 +432,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QPalette& arg0 = getqtype<QPaletteType>(param_this);
         const QPalette arg1 = getqtype<QPaletteType>(param_p);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QPalette_operatorEQ_EQ__bool_QPalette_QPalette(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_p)
@@ -440,7 +440,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QPalette& arg0 = getqtype<QPaletteType>(param_this);
         const QPalette arg1 = getqtype<QPaletteType>(param_p);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QPalette0, Pointer)
diff --git i/src/lib/mu/MuQt6/QProcessEnvironmentType.cpp w/src/lib/mu/MuQt6/QProcessEnvironmentType.cpp
index 3a6f999..dae0ab9 100644
--- i/src/lib/mu/MuQt6/QProcessEnvironmentType.cpp
+++ w/src/lib/mu/MuQt6/QProcessEnvironmentType.cpp
@@ -186,7 +186,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QProcessEnvironment& arg0 = getqtype<QProcessEnvironmentType>(param_this);
         const QProcessEnvironment arg1 = getqtype<QProcessEnvironmentType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QProcessEnvironment_operatorEQ_EQ__bool_QProcessEnvironment_QProcessEnvironment(Mu::Thread& NODE_THREAD, Pointer param_this,
@@ -195,7 +195,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QProcessEnvironment& arg0 = getqtype<QProcessEnvironmentType>(param_this);
         const QProcessEnvironment arg1 = getqtype<QProcessEnvironmentType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QProcessEnvironment_systemEnvironment_QProcessEnvironment(Mu::Thread& NODE_THREAD)
diff --git i/src/lib/mu/MuQt6/QRegionType.cpp w/src/lib/mu/MuQt6/QRegionType.cpp
index 3c6b090..757666f 100644
--- i/src/lib/mu/MuQt6/QRegionType.cpp
+++ w/src/lib/mu/MuQt6/QRegionType.cpp
@@ -268,7 +268,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QRegion& arg0 = getqtype<QRegionType>(param_this);
         const QRegion arg1 = getqtype<QRegionType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     Pointer qt_QRegion_operatorAmp__QRegion_QRegion_QRegion(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_r)
@@ -340,7 +340,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QRegion& arg0 = getqtype<QRegionType>(param_this);
         const QRegion arg1 = getqtype<QRegionType>(param_r);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QRegion_operatorCaret__QRegion_QRegion_QRegion(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_r)
diff --git i/src/lib/mu/MuQt6/QRegularExpressionType.cpp w/src/lib/mu/MuQt6/QRegularExpressionType.cpp
index a5ecae1..ce09a73 100644
--- i/src/lib/mu/MuQt6/QRegularExpressionType.cpp
+++ w/src/lib/mu/MuQt6/QRegularExpressionType.cpp
@@ -173,7 +173,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QRegularExpression& arg0 = getqtype<QRegularExpressionType>(param_this);
         const QRegularExpression arg1 = getqtype<QRegularExpressionType>(param_re);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QRegularExpression_operatorEQ_EQ__bool_QRegularExpression_QRegularExpression(Mu::Thread& NODE_THREAD, Pointer param_this,
@@ -182,7 +182,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QRegularExpression& arg0 = getqtype<QRegularExpressionType>(param_this);
         const QRegularExpression arg1 = getqtype<QRegularExpressionType>(param_re);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QRegularExpression_anchoredPattern_string_string(Mu::Thread& NODE_THREAD, Pointer param_expression)
diff --git i/src/lib/mu/MuQt6/QTextBlockType.cpp w/src/lib/mu/MuQt6/QTextBlockType.cpp
index 763b0ca..aadcd88 100644
--- i/src/lib/mu/MuQt6/QTextBlockType.cpp
+++ w/src/lib/mu/MuQt6/QTextBlockType.cpp
@@ -240,7 +240,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTextBlock arg0 = getqtype<QTextBlockType>(param_this);
         const QTextBlock arg1 = getqtype<QTextBlockType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QTextBlock_operatorLT__bool_QTextBlock_QTextBlock(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -248,7 +248,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTextBlock arg0 = getqtype<QTextBlockType>(param_this);
         const QTextBlock arg1 = getqtype<QTextBlockType>(param_other);
-        return arg0.operator<(arg1);
+        return arg0 < arg1;
     }
 
     bool qt_QTextBlock_operatorEQ_EQ__bool_QTextBlock_QTextBlock(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -256,7 +256,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTextBlock arg0 = getqtype<QTextBlockType>(param_this);
         const QTextBlock arg1 = getqtype<QTextBlockType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_blockFormatIndex0, int)
diff --git i/src/lib/mu/MuQt6/QTextCursorType.cpp w/src/lib/mu/MuQt6/QTextCursorType.cpp
index 4766eca..6ce8ffa 100644
--- i/src/lib/mu/MuQt6/QTextCursorType.cpp
+++ w/src/lib/mu/MuQt6/QTextCursorType.cpp
@@ -429,7 +429,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QTextCursor& arg0 = getqtype<QTextCursorType>(param_this);
         const QTextCursor arg1 = getqtype<QTextCursorType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QTextCursor_operatorLT__bool_QTextCursor_QTextCursor(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -437,7 +437,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QTextCursor& arg0 = getqtype<QTextCursorType>(param_this);
         const QTextCursor arg1 = getqtype<QTextCursorType>(param_other);
-        return arg0.operator<(arg1);
+        return arg0 < arg1;
     }
 
     bool qt_QTextCursor_operatorLT_EQ__bool_QTextCursor_QTextCursor(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -445,7 +445,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QTextCursor& arg0 = getqtype<QTextCursorType>(param_this);
         const QTextCursor arg1 = getqtype<QTextCursorType>(param_other);
-        return arg0.operator<=(arg1);
+        return arg0 <= arg1;
     }
 
     bool qt_QTextCursor_operatorEQ_EQ__bool_QTextCursor_QTextCursor(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -453,7 +453,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QTextCursor& arg0 = getqtype<QTextCursorType>(param_this);
         const QTextCursor arg1 = getqtype<QTextCursorType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     bool qt_QTextCursor_operatorGT__bool_QTextCursor_QTextCursor(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -461,7 +461,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QTextCursor& arg0 = getqtype<QTextCursorType>(param_this);
         const QTextCursor arg1 = getqtype<QTextCursorType>(param_other);
-        return arg0.operator>(arg1);
+        return arg0 > arg1;
     }
 
     bool qt_QTextCursor_operatorGT_EQ__bool_QTextCursor_QTextCursor(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -469,7 +469,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QTextCursor& arg0 = getqtype<QTextCursorType>(param_this);
         const QTextCursor arg1 = getqtype<QTextCursorType>(param_other);
-        return arg0.operator>=(arg1);
+        return arg0 >= arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QTextCursor0, Pointer)
diff --git i/src/lib/mu/MuQt6/QTimeZoneType.cpp w/src/lib/mu/MuQt6/QTimeZoneType.cpp
index aa437af..e966461 100644
--- i/src/lib/mu/MuQt6/QTimeZoneType.cpp
+++ w/src/lib/mu/MuQt6/QTimeZoneType.cpp
@@ -236,7 +236,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTimeZone arg0 = getqtype<QTimeZoneType>(param_this);
         const QTimeZone arg1 = getqtype<QTimeZoneType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QTimeZone_operatorEQ_EQ__bool_QTimeZone_QTimeZone(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -244,7 +244,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTimeZone arg0 = getqtype<QTimeZoneType>(param_this);
         const QTimeZone arg1 = getqtype<QTimeZoneType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QTimeZone_fromDurationAheadOfUtc_QTimeZone_int(Mu::Thread& NODE_THREAD, int param_offset)
diff --git i/src/lib/mu/MuQt6/QTransformType.cpp w/src/lib/mu/MuQt6/QTransformType.cpp
index dc1bdca..7a36d6a 100644
--- i/src/lib/mu/MuQt6/QTransformType.cpp
+++ w/src/lib/mu/MuQt6/QTransformType.cpp
@@ -408,7 +408,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTransform arg0 = getqtype<QTransformType>(param_this);
         const QTransform arg1 = getqtype<QTransformType>(param_matrix);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     Pointer qt_QTransform_operatorStar__QTransform_QTransform_QTransform(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_matrix)
@@ -448,7 +448,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         QTransform arg0 = getqtype<QTransformType>(param_this);
         const QTransform arg1 = getqtype<QTransformType>(param_matrix);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QTransform_fromScale_QTransform_double_double(Mu::Thread& NODE_THREAD, double param_sx, double param_sy)
diff --git i/src/lib/mu/MuQt6/QUrlQueryType.cpp w/src/lib/mu/MuQt6/QUrlQueryType.cpp
index 1a69e2f..09a877e 100644
--- i/src/lib/mu/MuQt6/QUrlQueryType.cpp
+++ w/src/lib/mu/MuQt6/QUrlQueryType.cpp
@@ -205,7 +205,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QUrlQuery& arg0 = getqtype<QUrlQueryType>(param_this);
         const QUrlQuery arg1 = getqtype<QUrlQueryType>(param_other);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QUrlQuery_operatorEQ_EQ__bool_QUrlQuery_QUrlQuery(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_other)
@@ -213,7 +213,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QUrlQuery& arg0 = getqtype<QUrlQueryType>(param_this);
         const QUrlQuery arg1 = getqtype<QUrlQueryType>(param_other);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     static NODE_IMPLEMENTATION(_n_QUrlQuery0, Pointer)
diff --git i/src/lib/mu/MuQt6/QUrlType.cpp w/src/lib/mu/MuQt6/QUrlType.cpp
index ae4feb7..cbce120 100644
--- i/src/lib/mu/MuQt6/QUrlType.cpp
+++ w/src/lib/mu/MuQt6/QUrlType.cpp
@@ -435,7 +435,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QUrl& arg0 = getqtype<QUrlType>(param_this);
         const QUrl arg1 = getqtype<QUrlType>(param_url);
-        return arg0.operator!=(arg1);
+        return arg0 != arg1;
     }
 
     bool qt_QUrl_operatorEQ_EQ__bool_QUrl_QUrl(Mu::Thread& NODE_THREAD, Pointer param_this, Pointer param_url)
@@ -443,7 +443,7 @@ namespace Mu
         MuLangContext* c = static_cast<MuLangContext*>(NODE_THREAD.context());
         const QUrl& arg0 = getqtype<QUrlType>(param_this);
         const QUrl arg1 = getqtype<QUrlType>(param_url);
-        return arg0.operator==(arg1);
+        return arg0 == arg1;
     }
 
     Pointer qt_QUrl_fromEncoded_QUrl_QByteArray_int(Mu::Thread& NODE_THREAD, Pointer param_input, int param_parsingMode)
diff --git i/src/lib/mu/MuQt6/SignalSpy.cpp w/src/lib/mu/MuQt6/SignalSpy.cpp
index a6b5807..b028259 100644
--- i/src/lib/mu/MuQt6/SignalSpy.cpp
+++ w/src/lib/mu/MuQt6/SignalSpy.cpp
@@ -278,5 +278,4 @@ namespace Mu
 
 //  hijack qt_metacall by using a filtered version of the moc file
 //  that has the function renamed to original_qt_metacall
-#include <MuQt6/generated/moc_SignalSpy_filtered.hpp>
 } // namespace Mu
