diff --git a/cmake/macros/rv_quote_file.cmake b/cmake/macros/rv_quote_file.cmake
--- a/cmake/macros/rv_quote_file.cmake
+++ b/cmake/macros/rv_quote_file.cmake
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+FIND_PACKAGE(Python3 REQUIRED COMPONENTS Interpreter)
 FIND_PROGRAM(_quote_file quoteFile HINT ${PROJECT_SOURCE_DIR}/src/build/quoteFile.py NO_CACHE REQUIRED)
 
 FUNCTION(quote_files OUTPUT_LIST)
@@ -61,7 +62,7 @@ FUNCTION(quote_files OUTPUT_LIST)
     )
 
     MESSAGE(STATUS "Creating '${arg_SUFFIX}' file: '${_src_name}' --> '${_dest_name}'")
-    EXECUTE_PROCESS(COMMAND python3 ${_quote_file} "${_src_name}" "${_dest_name}" "${_stem}_${arg_SUFFIX}" COMMAND_ERROR_IS_FATAL ANY)
+    EXECUTE_PROCESS(COMMAND "${Python3_EXECUTABLE}" ${_quote_file} "${_src_name}" "${_dest_name}" "${_stem}_${arg_SUFFIX}" COMMAND_ERROR_IS_FATAL ANY)
 
     LIST(APPEND _cooked_list ${_dest_name})
   ENDFOREACH()
@@ -104,6 +105,6 @@ FUNCTION(quote_file OUTPUT_FILENAME)
   ADD_CUSTOM_COMMAND(
     OUTPUT ${_dest_name}
-    COMMAND python3 ${_quote_file} "${_src_name}" "${_dest_name}" "${arg_CPP_SYMBOL}"
+    COMMAND "${Python3_EXECUTABLE}" ${_quote_file} "${_src_name}" "${_dest_name}" "${arg_CPP_SYMBOL}"
     DEPENDS ${_src_name}
     COMMENT "Quoting '${arg_INPUT_FILENAME}' as '${_stem}.cpp'"
     VERBATIM
