diff --git a/src/bin/apps/rvprof/VisMainWindow.cpp b/src/bin/apps/rvprof/VisMainWindow.cpp
--- a/src/bin/apps/rvprof/VisMainWindow.cpp
+++ b/src/bin/apps/rvprof/VisMainWindow.cpp
@@ -10,7 +10,18 @@
 #ifdef PLATFORM_DARWIN
 #include <OpenGL/glu.h>
 #else
+#ifdef PLATFORM_WINDOWS
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#define RVPROF_DEFINED_WIN32_LEAN_AND_MEAN
+#endif
+#include <windows.h>
+#endif
 #include <GL/glu.h>
+#ifdef RVPROF_DEFINED_WIN32_LEAN_AND_MEAN
+#undef WIN32_LEAN_AND_MEAN
+#undef RVPROF_DEFINED_WIN32_LEAN_AND_MEAN
+#endif
 #endif

 using namespace TwkGLText;
