diff --git a/cmake/macros/rv_make_std_lib_name.cmake b/cmake/macros/rv_make_std_lib_name.cmake
--- a/cmake/macros/rv_make_std_lib_name.cmake
+++ b/cmake/macros/rv_make_std_lib_name.cmake
@@ -54,7 +54,6 @@ MACRO(RV_MAKE_STANDARD_LIB_NAME base_name version lib_type debug_postfix)
     SET(_implibpath
         ${_lib_dir}/${_implibname}
     )
-    LIST(APPEND _byproducts ${_implibpath})
   ELSE()
     SET(_libpath
         ${_lib_dir}/${_libname}
@@ -64,6 +63,9 @@ MACRO(RV_MAKE_STANDARD_LIB_NAME base_name version lib_type debug_postfix)
   SET(_byproducts
       ${_libpath}
   )
+  IF(RV_TARGET_WINDOWS)
+    LIST(APPEND _byproducts ${_implibpath})
+  ENDIF()
 
   MESSAGE(DEBUG "RV_MAKE_STANDARD_LIB_NAME:")
   MESSAGE(DEBUG "  _libname      ='${_libname}'")
