diff --git i/CMakeLists.txt w/CMakeLists.txt
index 5589a46..ff1677c 100644
--- i/CMakeLists.txt
+++ w/CMakeLists.txt
@@ -10,9 +10,9 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.27)
 # These variables are parsed by sphinx for the documentation. One-line formatting facilitates parsing and readability.
 # cmake-format: off
 SET(RV_MAJOR_VERSION "3" CACHE STRING "RV's version major")
-SET(RV_MINOR_VERSION "1" CACHE STRING "RV's version minor")
+SET(RV_MINOR_VERSION "2" CACHE STRING "RV's version minor")
 SET(RV_REVISION_NUMBER "0" CACHE STRING "RV's revision number")
-SET(RV_VERSION_YEAR "2025" CACHE STRING "RV's year of release.")
+SET(RV_VERSION_YEAR "2026" CACHE STRING "RV's year of release.")
 # cmake-format: on
 
 IF(NOT CMAKE_BUILD_TYPE)
diff --git i/cmake/defaults/CY2026.cmake w/cmake/defaults/CY2026.cmake
index d5a5da4..a0c5d75 100644
--- i/cmake/defaults/CY2026.cmake
+++ w/cmake/defaults/CY2026.cmake
@@ -42,21 +42,21 @@ IF(RV_VFX_PLATFORM STREQUAL "CY2026")
 
   # OCIO https://github.com/AcademySoftwareFoundation/OpenColorIO
   SET(RV_DEPS_OCIO_VERSION
-      "2.5.0"
+      "2.5.2"
   )
   SET(RV_DEPS_OCIO_VERSION_SHORT
       "2_5"
   )
   SET(RV_DEPS_OCIO_DOWNLOAD_HASH
-      "fd402ea99fd2c4e5b43ea31b4a3387df"
+      "5ad02299def485e65abb4199bb04bc58"
   )
 
   # OpenEXR https://github.com/AcademySoftwareFoundation/openexr/releases
   SET(RV_DEPS_OPENEXR_VERSION
-      "3.4.3"
+      "3.4.13"
   )
   SET(RV_DEPS_OPENEXR_DOWNLOAD_HASH
-      "c11676598aa27a01a1cd21ad75b72e44"
+      "9899ae9513a939c793d5b7663eda7b62"
   )
   SET(RV_DEPS_OPENEXR_LIBNAME_SUFFIX
       "3_4"
@@ -84,10 +84,10 @@ IF(RV_VFX_PLATFORM STREQUAL "CY2026")
 
   # PySide
   SET(RV_DEPS_PYSIDE_VERSION
-      "6.8.3"
+      "6.10.2"
   )
   SET(RV_DEPS_PYSIDE_DOWNLOAD_HASH
-      "2a81028f5896edeb9c2a80adac3a8e68"
+      "2c0c0cbb4cbc267a891b62d404d55bad"
   )
   SET(RV_DEPS_PYSIDE_TARGET
       "RV_DEPS_PYSIDE6"
@@ -106,7 +106,7 @@ IF(RV_VFX_PLATFORM STREQUAL "CY2026")
 
   # Qt
   SET(RV_DEPS_QT_VERSION
-      "6.8.3"
+      "6.10.2"
   )
   SET(RV_DEPS_QT_MAJOR
       "6"
diff --git i/cmake/defaults/CYCOMMON.cmake w/cmake/defaults/CYCOMMON.cmake
index e3a5514..25e0fa3 100644
--- i/cmake/defaults/CYCOMMON.cmake
+++ w/cmake/defaults/CYCOMMON.cmake
@@ -176,10 +176,10 @@ SET(RV_DEPS_JPEGTURBO_VERSION_LIB
 
 # oiio https://github.com/AcademySoftwareFoundation/OpenImageIO
 SET(RV_DEPS_OIIO_VERSION
-    "3.1.12.0"
+    "3.1.14.1"
 )
 SET(RV_DEPS_OIIO_DOWNLOAD_HASH
-    "4f7cb0e132117613d828439d987e187c"
+    "48f25c3588bab6adad68cb7d99dffc2a"
 )
 
 # openjpeg https://github.com/uclouvain/openjpeg
