diff --git a/src/cmake/add_oiio_plugin.cmake b/src/cmake/add_oiio_plugin.cmake
--- a/src/cmake/add_oiio_plugin.cmake
+++ b/src/cmake/add_oiio_plugin.cmake
@@ -82,2 +82,2 @@
-            target_link_libraries (${_plugin_NAME} PUBLIC OpenImageIO
-                                                   PRIVATE ${_plugin_LINK_LIBRARIES})
+            target_link_libraries (${_plugin_NAME}
+                                  PRIVATE OpenImageIO ${_plugin_LINK_LIBRARIES})
