diff --git a/src/cmake/add_oiio_plugin.cmake b/src/cmake/add_oiio_plugin.cmake
index 8d9d4f8..e5c7657 100644
--- a/src/cmake/add_oiio_plugin.cmake
+++ b/src/cmake/add_oiio_plugin.cmake
@@ -74,8 +74,8 @@ macro (add_oiio_plugin)
                                         OpenImageIO_EXPORTS)
             target_compile_options (${_plugin_NAME} PRIVATE ${_plugin_COMPILE_OPTIONS})
             target_include_directories (${_plugin_NAME} BEFORE PRIVATE ${_plugin_INCLUDE_DIRS})
-            target_link_libraries (${_plugin_NAME} PUBLIC OpenImageIO
-                                                   PRIVATE ${_plugin_LINK_LIBRARIES})
+            target_link_libraries (${_plugin_NAME}
+                                  PRIVATE OpenImageIO ${_plugin_LINK_LIBRARIES})
             set_target_properties (${_plugin_NAME} PROPERTIES PREFIX "" FOLDER "Plugins")
             install_targets (${_plugin_NAME})
         endif ()
