diff --git a/cmake/defaults/Packages.cmake b/cmake/defaults/Packages.cmake
--- a/cmake/defaults/Packages.cmake
+++ b/cmake/defaults/Packages.cmake
@@ -69,8 +69,12 @@ macro(setup_python_package package)
         # This option indicates that we don't want libraries to explicitly link
         # to the Python libraries. However, executables must link to the Python
         # libraries to avoid missing symbol errors. See BUILDING.md for details.
         if(PXR_PY_UNDEFINED_DYNAMIC_LOOKUP AND NOT WIN32)
-            set(PYTHON_LIBRARIES "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,EXECUTABLE>:${package}::Python>")
+            if(APPLE)
+                set(PYTHON_LIBRARIES "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,EXECUTABLE>:$<LINK_LIBRARY:NEEDED_LIBRARY,${package}::Python>>")
+            else()
+                set(PYTHON_LIBRARIES "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,EXECUTABLE>:${package}::Python>")
+            endif()
         else()
             set(PYTHON_LIBRARIES "${package}::Python")
         endif()
