diff --git a/cmake/libuhdr.pc.in b/cmake/libuhdr.pc.in
index 89e0cf5..5897860 100644
--- a/cmake/libuhdr.pc.in
+++ b/cmake/libuhdr.pc.in
@@ -5,6 +5,6 @@ Name: @PROJECT_NAME@
 Description: @CMAKE_PROJECT_DESCRIPTION@
 Version: @PROJECT_VERSION@
 Requires.private: libjpeg
-Cflags: -I${includedir}
+Cflags: -DUHDR_USING_SHARED_LIBRARY -I${includedir}
 Libs: -L${libdir} -l@UHDR_TARGET_NAME@
 Libs.private: @CMAKE_THREAD_LIBS_INIT@ @UHDR_GL_DEPS@
